src/pages/GroupSettings.tsx

```
@@ -0,0 +1,756 @@
+      
+      // Make sure the owner is included in the moderators list
+      if (!modPubkeys.includes(communityEvent.pubkey)) {
+        modPubkeys.push(communityEvent.pubkey);
+      }
+      
+      // Remove duplicates
+      const uniqueModPubkeys = [...new Set(modPubkeys)];
+      
+      setModerators(uniqueModPubkeys);
+    }
+  }, [community]);
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)